docs-src/typescript/clients.md

```diff
@@ -41,13 +41,15 @@ import { example } from './workflows';
 
 async function run() {
   // Connect to the default Server location (localhost:7233)
+  console.log(Date.now());
   const connection = await Connection.connect();
   // In production, pass options to configure TLS and other settings:
   // {
   //   address: 'foo.bar.tmprl.cloud',
   //   tls: {}
   // }
 
+  console.log(Date.now());
```

Comment:
This and several other changes that merging this PR will make to the main branch don't seem related to the PR title/description